src/js/src/components/inlinePlayer/InlinePlayer.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div>
+    <video
+      ref="inlinePlayerObj"
+      class="video-js" />
+  </div>
+</template>
+
+<style lang="scss">
+@import '../../../node_modules/video.js/dist/video-js.min.css';
+</style>
+
+<script>
+import videojs from 'video.js'
+import {mapActions} from 'vuex'
+
+export default {
+    name: 'InlinePlayer',
+    props: {
+        result: {
+            type: Object,
+            default() {
+                return {}
+            }
+        }
+    },
+
+    data() {
+        return {
+            playerInstance: null
+        }
+    },
+
+    mounted() {
+        this.playerInstance = videojs(this.$refs.inlinePlayerObj, this.getPlayerOptions())
+        this.playerInstance.on('error', () => {
+        const errorObj = this.player.error()
```

Comment:
I get an error here saying 

IDEOJS: ERROR: TypeError: Cannot read property 'error' of undefined
    at Player.eval (InlinePlayer.vue?aa9c:37)
    at HTMLDivElement.data.dispatcher (video.es.js?f0e2:1899)
    at trigger (video.es.js?f0e2:2035)
    at Player.trigger$1 (video.es.js?f0e2:2921)
    at Player.error (video.es.js?f0e2:24603)
    at Player.eval (video.es.js?f0e2:24035)
    at eval (video.es.js?f0e2:4626)

Reply:
To clarify, this happens when I try to play a video of the .wmv format.

Reply:
Even more clarification: I believe i should get an error in our notification system - that doesn't seem to happen.

Reply:
That's a nasty one....will look into this.

Reply:
Found it! And good catch btw 👍 

It was a refactoring mishap on my part.